### DISCUSSION TITLE
Bring back old homepage

### DISCUSSION THREAD

**Discussion by therealmate** [community/community#66169] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I can't see the commits from the repositories i follow with the new homepage

**Comment by Rainyan** [comment]
I suppose un-ticking every single feed filter accomplishes the task of not having unwanted crap on my front page, but the amount of wasted negative space is reaching tragic proportions.

**Comment by sgf** [comment]
The homepage of the new version is very confusing, and I can't see which valuable repositories have been starred by the people I follow.
I want to go back to the old home page.

**Comment by aaq1adqwe** [comment]
yes，fuckoff，

What a foolish decision！！！！

**Comment by sgf** [comment]
I don't know why in the programming or software industry.
I also found that many unprofessional people occupy highly professional positions.
In the C# community, Microsoft let a person who has no trace of writing code on the github page participate in managing the direction of the C# language.
In Github, it seems that another product manager who does not use github is found to design the homepage of github.
Why do these absurd things always happen in Microsoft's business lines?

**Comment by rlggyp** [comment]
I believe the homepage looks very clean and appealing at the moment. You should try the light mode instead.

**Comment by Itumadavid1** [comment]
If you're unable to see the commits from the repositories you follow on GitHub's new homepage, here are some steps you can take to troubleshoot and potentially resolve the issue:

1. **Check Your Following List:**
   - Make sure you are still following the repositories you want to see commits from. Sometimes, unintentional actions can lead to unfollowing repositories.

2. **Clear Browser Cache:**
   - Cached data in your web browser can sometimes cause display issues. Try clearing your browser cache and then reloading the GitHub homepage.

3. **Check GitHub Settings:**
   - Ensure that your GitHub settings are configured correctly. Go to your GitHub account settings, then to the "Notifications" section. Make sure that "Automatically watch repositories" is enabled if you want to receive notifications about commits in repositories you follow.

4. **Check Repository Notifications:**
   - In addition to following repositories, you can also "watch" repositories. This can be set on a per-repository basis. Go to the repository you're interested in, click on "Watch," and ensure that you're set to receive notifications for "Releases" and "Watching."

5. **Review Filters:**
   - GitHub's new homepage may have filter options to sort the feed. Make sure you haven't accidentally filtered out commits. Look for filter options like "Issues," "Pull Requests," or "Commits" and make sure they are appropriately selected.

6. **Use the "Following" Tab:**
   - On the new GitHub homepage, there are tabs for "Overview," "Repositories," and "Marketplace." Ensure you are on the "Overview" tab to see the feed of activity from those you follow.

7. **Check for Browser Extensions:**
   - Browser extensions or add-ons can sometimes interfere with the display of web pages. Try disabling any browser extensions that might affect your GitHub experience to see if that resolves the issue.

8. **Update Browser:**
   - Ensure your web browser is up to date. Outdated browsers may not display web pages correctly.

9. **Contact GitHub Support:**
   - If the issue persists after trying these steps, it's a good idea to contact GitHub Support. They can provide more specific assistance and investigate whether there might be an issue on their end.

Remember that GitHub periodically updates its user interface and features, so it's possible that changes in the interface have affected how your feed is displayed. If you're experiencing a persistent issue, GitHub Support is your best resource for resolving it.

**Reply by port19x** [reply]
Chatgpt very much? Or traditional copy pasta?

**Comment by wangrongding** [comment]
# Github Old Feed  

💪 Using this script you can restore the old version of the feed.
https://github.com/wangrongding/github-old-feed  

<table>
  <tr>
      <td width="50%" align="center"><b>Before</b></td>
      <td width="50%" align="center"><b>After use</b></td>
  </tr>
  <tr>
     <td><img src="https://assets.fedtop.com/picbed/1694066442147.jpg"/></td>
     <td><img src="https://assets.fedtop.com/picbed/1694066437621.jpg"/></td>
  </tr>
</table>
